### DISCUSSION TITLE
Google icon missing in @Mantine/ds

### DISCUSSION THREAD

**Discussion by SebasQuirogaUCP** [mantinedev/mantine#4742] [Feature requests]
I noticed that `@mantine-ds` package includes quite a variety of social icons:

* Discord icon
* Github icon
* Twitter icon
* YarnI icon
* Npm icon

But it lacks of some others like Google icon.

I would be pleased to open a PR adding it.

**Comment by rtivital** [comment]
`@mantine/ds` package includes only icons used on *.mantine.dev websites, it is not designed to be used as an icon library

**Reply by SebasQui** [reply]
Thanks for your reply @rtivital !

Do we have a central place for icons?
That would be a really nice-to-have.

Regards.

**Reply by rtivital** [reply]
It is out of the project scope, you can use any available icons library with Mantine

**Reply by SebasQui** [reply]
You are right.
Thank you for your reply.
